src/Compilers/CSharp/Portable/Lowering/SynthesizedMethodBaseSymbol.cs

```diff
@@ -147,6 +149,8 @@ private ImmutableArray<ParameterSymbol> MakeParameters()
         /// </summary>
         internal virtual bool InheritsBaseMethodAttributes => false;
 
+        protected virtual bool InheritsParams => false;
```

Comment:
Consider using `abstract` instead, so it's clearer which implementations what behavior